js_tests/admin/DateTimeShortcuts.test.js

```diff
@@ -40,3 +40,9 @@ QUnit.test('time zone offset warning', function(assert) {
     $('body').attr('data-admin-utc-offset', savedOffset);
     assert.equal($('.timezonewarning').text(), 'Note: You are 1 hour behind server time.');
 });
+
+QUnit.test('calendar escape format', function(assert) {
+    const date_input_format = "'%d \r\t\\\n %B \n\\\r\t %Y'";
+    format = DateTimeShortcuts.escapeFormat(date_input_format);
+    assert.equal(format, "\\n February \\n 12 \\n 1998 \\r \\r \\r \\t \\t \\t \\\\ \\\\ \\' \\'");
```

Comment:
Why are the tests green? Shouldn't this fail as we didn't update the expected string 🤔

Reply:
It fails as expected. You're a first-time contributor so we need to manually approve each run.